stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -123,13 +135,200 @@ The `description` property allows you to provide additional information. It can
   </Story>
 </Canvas>
 
-## Right and left elements
+## Grid layout
+
+If used within a List component the style of the listItem can be changed to a grid style.
```

Comment:
I think the overall explanation here needs to be modified to make it more fluid and clear. And be careful with the typography :
 `that it the`=> `that if the`
`the selectionned layout` => `the selected`

Reply:
I agree technically even without `List` the style can be changed by passing a `layout` value in the `ListItem`